beta/tests/tensorflow/test_models/mobilenet_v3.py

```diff
@@ -11,208 +11,8 @@
  limitations under the License.
 """
 
-from tensorflow.python.keras import backend
-from tensorflow.python.keras import models
-from tensorflow.python.keras.applications import imagenet_utils
-from tensorflow.python.keras import layers
+from beta.examples.tensorflow.common.models import MobileNetV3Small as mobilenet_v3_small
 
 
 def MobileNetV3Small(input_shape=None):
```

Comment:
I think we can remove this file. @alexsu52, do you agree with me?

Reply:
Test failed without it

Reply:
@andrey-churkin essentially, we can remove this file and import the model from the example directly. But I hope TF will fix the issue with this model and we will add the correct implementation to the test. I would leave this file in the tests.